src/modal/modal.scss

```
@@ -67,11 +78,52 @@
         margin-right: $iui-s;
       }
     }
+
+    .iui-modal-content {
+      flex-grow: 2;
+      margin: 0 -#{$iui-m};
+      padding: 0 $iui-m;
+      overflow-y: auto;
+      @supports (overflow-y: overlay) {
+        overflow-y: overlay;
+      }
```

There's no need to wrap this in `@supports`. It will be ignored in browsers that don't support it

```suggestion
      overflow-y: overlay;
```

---

Done. So `@supports` doesn't need to be used if the condition is exactly the same as the body?